crates/lgn-api-codegen/src/python/templates/models.py.jinja

```diff
@@ -99,7 +100,60 @@ class {{ model_name }}:
             self.{{ field.name }},
         {%- endfor %}
         )
-    
+
+{% when Type::StructMap with { fields, map } %}
+class {{ model_name }}:
+    def __init__(
+        self,
+        {%- for field in fields.values() %}
+        {{ field.name }}, # : {{ field.type_|fmt_type(ctx) }} {% if let Some(description) = field.description -%} # {{ description }}{%- endif %}
+        {%- endfor %}
+        additional_properties: dict[str, {{ map|fmt_type(ctx) }}]
```

Comment:
additional_properties are included to the class but not used when ser/de the request/response @kirdaybov

Reply:
I've added an issue to add support for it https://github.com/legion-labs/legion/issues/2050